.github/workflows/_win-build.yml

```diff
@@ -180,6 +191,14 @@ jobs:
           name: ${{ inputs.build-environment }}
           path: C:\${{ github.run_id }}\build-results
 
+      - name: Configure AWS Credentials - Upload sccache
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
As per your pr description, are these steps intended to be cloud agnostic, so they'll succeed even if the job is running on a non-aws machine?

Reply:
Yes, that's right. These configure-aws-credentials should work in any cloud as long as OIDC connection is setup for whatever clouds we end up supporting. This removes any assumptions that the runner itself has permissions directly.